mutable_tree.go

```diff
@@ -397,7 +397,8 @@ func (tree *MutableTree) deleteVersionsFrom(version int64) error {
 			return cmn.NewError("cannot delete latest saved version (%d)", version)
 		}
 		if _, ok := tree.versions[version]; !ok {
-			return cmn.ErrorWrap(ErrVersionDoesNotExist, "")
+			//return cmn.ErrorWrap(ErrVersionDoesNotExist, "")
```

Comment:
the change looks fine to me, just out of curious, in what case iris using this method?

the calling tree is:
`LoadVersionForOverwriting` -> `deleteVersionsFrom`

Reply:
In iris, the prune strategy will only keep the latest 100 version and every 10000 height a version for older height. When we try to export state at very old height, the version of this height may have been deleted, so we have to rebuild the version. Before building the version, we should delete all version from the height to the latest height. Then this code will be triggered.

Reply:
but we can catch that error `ErrVersionDoesNotExist ` in the caller